src/manifolds/StiefelSubmersionMetric.jl

```diff
@@ -0,0 +1,290 @@
+@doc raw"""
+    StiefelSubmersionMetric{T<:Real} <: RiemannianMetric
+
+The submersion (or normal) metric family on the [`Stiefel`](@ref) manifold.
+
+# Constructor
+
+    StiefelSubmersionMetric(α)
+
+Construct the submersion metric on the Stiefel manifold with the parameter ``α > -1``.
+
+The submersion metric family has two special cases:
+- ``α = -\frac{1}{2}``: [`EuclideanMetric`](@ref)
+- ``α = 0``: [`CanonicalMetric`](@ref)
+"""
```

Comment:
Maybe we could add a reference to Zimmermann Hüper here?

Reply:
Yup! Documentation is still a WIP.